From f0476416c9b4c68cbb839a1e80ec49bc7c64d1c1 Mon Sep 17 00:00:00 2001
From: Yi-Chia Hsieh <Yi-Chia.Hsieh@mediatek.com>
Date: Tue, 26 Aug 2025 15:41:45 -0700
Subject: [PATCH 122/128] mtk: mac80211: rename IEEE80211_TWT_CONTROL_RESP_MODE

This bit is used to represent whether the reponsder sta may enter power save mode
outside of the TWT SPs. Rename to IEEE_TWT_CONTROL_RESPONDER_PM_MODE to
better represent the meaning of the bit.

Signed-off-by: Yi-Chia Hsieh <Yi-Chia.Hsieh@mediatek.com>
---
 include/linux/ieee80211-he.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/ieee80211-he.h b/include/linux/ieee80211-he.h
index e4084f2c..8882e987 100644
--- a/include/linux/ieee80211-he.h
+++ b/include/linux/ieee80211-he.h
@@ -19,7 +19,7 @@
 #include <linux/if_ether.h>
 
 #define IEEE80211_TWT_CONTROL_NDP			BIT(0)
-#define IEEE80211_TWT_CONTROL_RESP_MODE			BIT(1)
+#define IEEE80211_TWT_CONTROL_RESPONDER_PM_MODE		BIT(1)
 #define IEEE80211_TWT_CONTROL_NEG_TYPE_BROADCAST	BIT(3)
 #define IEEE80211_TWT_CONTROL_RX_DISABLED		BIT(4)
 #define IEEE80211_TWT_CONTROL_WAKE_DUR_UNIT		BIT(5)
-- 
2.45.2

